requirements.txt

```diff
@@ -0,0 +1,32 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile
```

Comment:
Will `pip-compile` reinstall those, or do I need to run this in a fresh venv before release time to regenerate and test against latest versions?

Reply:
`pip-compile --upgrade` will recreate it using the latest detected PyPI versions, `pip-compile` prefers versions currently listed in requirements.txt

Reply:
OK, thanks